src/main/java/model/Subject.java

```diff
@@ -0,0 +1,16 @@
+package model;
+
+import lombok.Data;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
```

Comment:
What the difference between Entity and Table?

Reply:
@javax.persistence.Entity: Specifies that the class is an entity. This annotation is applied to the entity class.

@javax.persistence.Table: Specifies the primary table for the annotated entity. Additional tables may be specified using SecondaryTable or SecondaryTables annotation. If no Table annotation is specified for an entity class, the default values apply.

https://34codefactory.wordpress.com/2021/01/08/difference-between-entity-and-table-annotation-code-factory/